container-images/pyspark-slim/setup_spark.py

```diff
@@ -47,7 +47,9 @@ def get_latest_spark_version() -> str:
         if ref.startswith("spark-") and "incubating" not in ref and "preview" not in ref
     ]
     # Compare versions semantically
-    latest_version = max(stable_versions, key=lambda ver: [int(sub_ver) for sub_ver in ver.split(".")])
+    latest_version = max(
```

Comment:
Why are there changes in this file? Our setup (ruff) should ensure that everyone formats files the same way.